modules/auxiliary/scanner/ssh/ssh_login_pubkey.rb

```diff
@@ -68,11 +68,11 @@ def ip
     datastore['RHOST']
   end
 
-  def session_setup(result, ssh_socket, fingerprint)
-    return unless ssh_socket
+  def session_setup(result, scanner.ssh_socket, fingerprint)
```

Comment:
I think this might actually be a typo since its calling the :ssh_socket method below anyway. Should probably be:
```
def session_setup(result, scanner, fingerprint)
```
or go back to passing the socket _to_ the :session_setup method and just referencing the socket in there since the scanner context isnt relevant to the session being set up on an endpoint for which it found valid creds.